dist/README.UPDATERS

```diff
@@ -1,3 +1,48 @@
+For Updaters to OBS 2.7 from OBS 2.6
+====================================
+
+WARNING: Please be aware, that this section is work in progress
+
+  1. Remove the OBS 2.6 Repository
+
+        zypper rs OBS:Server:2.6
+
+  2. Add the OBS 2.7 Repository and update repository cache
+
+        zypper ar http://download.opensuse.org/repositories/OBS:/Server:/2.7/openSUSE_13.2/OBS:Server:2.7.repo
+        zypper ref -s
+
+  3. Remove packages which might case conflicts
+
+        zypper -n remove ruby2.1-rubygem-passenger
+
+  4. Update packages
+
+        zypper dup --no-recommends --from OBS_Server_2.7
+
+  5. Update system defaults to use new ruby version
+
+        update-alternatives --set rake /usr/bin/rake.ruby.ruby2.3
+
+  6. Migrate database
+
+        cd /srv/www/obs/api/
+        RAILS_ENV="production" rake db:migrate
```

Comment:
use "rake.ruby2.3" here?

It is up to the user to select which rake he wants by default and rest of ruby must stay on 2.1 for yast ...